src/main/java/com/altenheim/kalender/controller/logicController/InitialSetupController.java

```diff
@@ -0,0 +1,74 @@
+package com.altenheim.kalender.controller.logicController;
+
+import com.altenheim.kalender.controller.logicController.SecureAesController;
+import com.altenheim.kalender.controller.viewController.PopupViewsController;
+import com.altenheim.kalender.interfaces.IIOController;
+import com.altenheim.kalender.interfaces.IWebsiteScraperController;
+import com.altenheim.kalender.models.SettingsModel;
+
+
```

Comment:
Neue Klasse die später alles aus dem Speicher aufrufen soll, bevor die Stage zum ersten Mal gerendert wird, damit die Properties im Settingsmodel belegt und somit alle Settings verfügbar sind ohne diese jedes Mal zur Laufzeit deserialisieren zu müssen. Wenn Settings nicht gefunden werden, kann der Nutzer initial Daten festlegen, exemplarisch ist das hier mit der Passwortabfrage so. Wenn die einmal erfolgreich war wird der Hash zum entschlüsseln des API Keys gespeichert und beim nächsten Programmstart ohne eine erneute Abfrage geladen. Hier kommt dann noch einiges dazu (UI Settings, gespeicherte Kontakte, Mail des Users, Kalender usw.) Wird alles mit dem IOController geladen, aber in dieser Klasse überprüft ob es da ist und überhaupt geladen werden kann.